test/Sia.Gateway.Tests/GlobalSuppressions.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+// This file is used by Code Analysis to maintain SuppressMessage 
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given 
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage(
+    "Design",
+    "CA1032:Implement standard exception constructors",
+    Justification = "Custom exception types with rigid usage requirements",
+    Scope = "namespace",
+    Target = "~T:Sia.Gateway.Tests.Middleware")]
```

Comment:
Per the comment farther below, I think one can open the scope up to the whole `module` and that'd prevent subsequent warnings about exception constructors:
```
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage(
    "Design",
    "CA1032:Implement standard exception constructors",
    Justification = "Custom exception types with rigid usage requirements",
    Scope = "module"
)]
```